### PR TITLE
Seperate Out Watchdog Msgs

### PR DESCRIPTION
**Pull request by vinnnyr**
## Issue:
Addresses #7 

## Description 
Separate out the `sw_watchdog_msgs` into it's own package so that these messages can be used in other packages

## Testing Method
- ROS2 Foxy
- Ubuntu / POP OS 20.04
- x86
- ran all launch files
- verified messages hsow up when invoking `ros2 interface show`
- created a test package with a `CMakeLists.txt` that looks like this. The `typesupport` issue no longer occurs when using `find_package()` from an external package

**Snippet of Test pkg CMakeLists**
```
# find dependencies
find_package(ament_cmake REQUIRED)
find_package(sw_watchdog_msgs REQUIRED)
```

### CODE REVIEW

File: sw_watchdog_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
+find_package(lifecycle_msgs REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
```
Comment (togaen) [inline sw_watchdog_msgs/CMakeLists.txt:15]: should probably remove dependencies on `rclcpp_lifecycle`, `lifecycle_msgs`, `rclcpp`, and `rclcpp_components` from the cmake file as well as the package manifest; they are not needed for this package.

Comment (vinnnyr) [reply sw_watchdog_msgs/CMakeLists.txt:15]: Yeah good point. The should be removed from `CMakeLists` and `pacakge.xml` now.

**Comment by vinnnyr** [comment]
Hi @togaen , can you please take another look?

**Comment by togaen** [comment]
@vinnnyr Yes, very sorry for the delay, I'll have a review done today.

**Review by togaen — DISMISSED** [review]
lgtm

**Comment by togaen** [comment]
@vinnnyr Are you able to turn on signed commits? We require those when merging code in. Otherwise, the PR looks good, thanks for contributing :-).

**Comment by vinnnyr** [comment]
Ok... looking into this. Will ping you here once I have figured it out. Thanks!

**Comment by togaen** [comment]
> Ok... looking into this. Will ping you here once I have figured it out. Thanks!

Great. Let me know if you have difficulties; we'll get your changes in one way or another.

**Comment by vinnnyr** [comment]
@togaen Everything should be signed now, please check when you get a free chance. Thanks for looking over this and incorporating it.

**Review by togaen — APPROVED** [review]

**Pull request merged**